cmd/web/main.go

```diff
@@ -74,6 +74,7 @@ func main() {
 	api.GET("/resourcegroups/:accountID", server.ResourceGroupHandler)
 	api.GET("/iam/accessGroups/:accountID", server.AccessGroupsHandler)
 	api.PUT("/iam/accessGroups/:accessGroupID/members", server.AddMemberHandler)
+	api.GET("/iam/accessGroups/:accessGroupID/members/:iamID", server.MembershipCheckHandler)
```

Comment:
the `iamID` here is the `IBMID` right?

Reply:
Should be IAM ID. IBM ID is their email used to log in.

IAM ID = IBMid-3100003HSP 
IBM ID = oliverd.rodriguez@gmail.com